RELEASE.md

```diff
@@ -23,11 +24,11 @@ The development process follows a standard GitHub workflow.
 
 ![Development Graph](.images/release-1.png)
 
-The main development is happening in `master` branch.
+The main development is happening in the `v2` branch.
 This is where new features land through Pull Requests.
-When all features for new release are merged, for example `v0.1.0`, we cut a branch for `release-0.1`.
+When all features for a new release have been merged, for example `v0.1.0`, we cut a branch for that release, e.g. `release-0.1`.
 
-The `release-0.1` branch will be base for `v0.1.0` release as well as the release candidates, e.g. `v0.1.0-rcX` and successive patch releases `v0.1.X`.
+The `release-0.1` branch will be the base for the `v0.1.0` release as well as the release candidates, i.e. `v0.1.0-rcX`, and successive patch releases, i.e. `v0.1.X`.
```

Comment:
meaning of e.g. is "for example" - i.e. stands for "in other words" which I think does not apply here

Reply:
Though....

Reply:
I think this is accurate because it's saying this is the form those things would take.
With that said, I'm happy to change both to `e.g.` if you find that less jarring :slightly_smiling_face:

Reply:
No, you are right. `i.e.` works here better :)